modules/khramov_i_lab2/test/test_khramov_ivan_radix.cpp

```diff
@@ -0,0 +1,103 @@
+// Copyright 2024 Khramov Ivan
+
+#include <gtest/gtest.h>
+#include <algorithm>
+
+#include "include/radix_sort.h"
+
+TEST(Khramov_Ivan_RadixTest, Check_Simple_Sort) {
+    // Arrange
+    std::vector<int> data {170, 45, 75, 90, 802, 24, 2, 66};
+    std::vector<int> expected = data;
+    // Act
+    std::sort(expected.begin(), expected.end());
+    RadixSort::radixSort(data);
+    // Assert
+    EXPECT_EQ(data, expected);
+}
+
+TEST(Khramov_Ivan_RadixTest, Check_Descending_Sort) {
+    // Arrange
+    std::vector<int> data {170, 45, 75, 90, 802, 24, 2, 66};
+    std::vector<int> expected = data;
+    // Act
+    std::sort(expected.begin(), expected.end(),
+     [](int a, int b) { return a > b; });
```

Comment:
```suggestion
    std::sort(expected.begin(), expected.end(), std::greater<int>());
```
Также для гарантии подключите `<functional>`
Поправьте и в других тестах

Reply:
Изменил